spin/cmds/pip.py

```diff
@@ -36,6 +36,11 @@ def install(pip_args, verbose, editable):
     pip_args += ["--no-build-isolation"]
     if editable:
         pip_args += ["--editable"]
+        if verbose:
```

Comment:
Thanks for this PR!

I find it a tiny bit weird to mix two slightly different things:
- verbose for pip commands, I want to see the output of the `pip` command
- verbose for meson editable install: I want to see the compilation steps when importing the package

Maybe I want to see the compilation output the first time (pip verbose) but not on import (meson editable verbose)

A separate parameter like `--meson-editable-verbose` would maybe makes sense?

Reply:
Sure, let's split the options.

Reply:
OK, done.